hexa/datasets/queue.py

```diff
@@ -1,18 +1,81 @@
 from logging import getLogger
 
+import pandas as pd
+from django.conf import settings
 from dpq.queue import AtLeastOnceQueue
 
-from hexa.datasets.models import DatasetFileMetadataJob
+from hexa.datasets.api import generate_download_url
+from hexa.datasets.models import (
+    DatasetFileMetadata,
+    DatasetFileMetadataJob,
+    DatasetVersionFile,
+)
 
 logger = getLogger(__name__)
 
 
+def read_file_content(download_url: str, content_type: str) -> pd.DataFrame:
```

Comment:
I do not think we need this function or at least to directly use the `DatasetVersionFile`

Reply:
And maybe rename it to `get_dataframe` or something more self-explanatory to what your returns no ?